src/modules/dashboard/staff/components/staff-popup.vue

```diff
@@ -94,12 +94,13 @@ const confirmButtonLabel = computed(() =>
     : i18n.global.t('Edit'),
 );
 
-const staffForm = ref<InstanceType<typeof StaffForm>>();
+const staffForm2 = ref<InstanceType<typeof StaffForm>>();
```

Comment:
Kurde, ta dwójeczka trochę tutaj boli, z jakiego powodu to musi być tak oznaczone? Nie mówię że teraz tak jest, ale zwykle to zwiastuje problemy w przyszłości z utzrymaniem, zrozumieniem rpzez innych devów itd.